src/__snapshots__/build.test.js.snap

```diff
@@ -47,3 +47,11 @@ Object {
   "uri": "https://pek3a.qingstor.com/test_bucket/test_object.jpg?acl&upload_id=test_upload_id",
 }
 `;
+
+exports[`parseVirtualHostRequestURI 1`] = `
```

Comment:
These files should be removed from git repo.

Reply:
https://jestjs.io/docs/en/snapshot-testing